packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -242,6 +244,30 @@ export const BootPopups = (): ReactElement => {
     user,
   ]);
 
+  /**
+   * Received gift plus modal
+   */
+  useEffect(() => {
+    if (!alerts?.shouldShowGiftPlus || !user.isPlus) {
```

Comment:
don't we use boot for this instead of alerts or just was not updated yet?

Reply:
I added it inside the `alerts` object as we did for `shouldShowFeedFeedback`

https://github.com/dailydotdev/daily-api/blob/main/src/routes/boot.ts#L542

Reply:
you can use it directly on `user` from `useAuthContext`, it will be confusing to put something that is not an alert into alerts object, it can also create invalid gql calls to alerts update mutation

Reply:
my bad I completely missed you added it there in the API PR

Reply:
Okay, so I've to add to the graphorm user object? @capJavert

Reply:
nvm added to user in boot